test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const urlBase = 'https://api.github.com';
+const Access = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+const query = {
+  description: 'Example of a gist',
+  files: {
+    'README.md': {
+      content: 'This is a gist example'
+    }
+  }
+};
+describe('DELETE Consumption and resources doesn not exist', () => {
+  let createGist;
+  before(async () => {
+    createGist = await Access.post(`${urlBase}/gists`, query);
+  });
+  it('Create a gist with POST and Promises', () => {
+    expect(createGist.status).to.equal(StatusCodes.CREATED);
+    expect(createGist.data).to.containSubset({
+      description: query.description,
+      files: query.files,
```

Comment:
If you can, you can change this params extraction of the query with [destructuring](https://www.geeksforgeeks.org/destructuring-assignment-in-javascript/)
Example: { ...query, public: false }

Reply:
expect(createGist.data).to.containSubset(query); should work, remember rename query variable to something more readable.